web_monitoring/links_diff.py

```diff
@@ -1,8 +1,12 @@
 from bs4 import BeautifulSoup
-from .html_diff_render import diff_elements, get_title
+from collections import Counter
+from .differs import compute_dmp_diff
+from difflib import SequenceMatcher
+from .html_diff_render import get_title, _html_for_dmp_operation
+import re
 
 
-def links_diff(a_text, b_text):
+def links_diff(a_text, b_text, json=False):
     """
```

Comment:
Well… it’s a kwarg because this is the HTTP interface, too. Should I separate out a library-focused entry-point to this module from the diff *server* focused entry point?

Reply:
(Side note: would be great to get a standardized format param to these diffs… e.g. `/{diff name}.{format kwarg}` or maybe basing it off the `Accept` request header.)

Reply:
I guess we could just register two endpoints with two funcs, one at `/links` → `links_diff_html()` and one at `/links.json` → `links_diff.links_diff()`. Do you think that’s preferable?

Reply:
I like that better, yeah. Obviously shared code can go into a utility function if there is enough to justify one.

Reply:
@danielballan how’s this? https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/153/commits/33a1b0ead5a0a616027b0128b73967f0dc521142

Am I bending over too far backwards to accommodate a formatting that is only theoretical?

I made the routes `/links` and `/links_json` (instead of `/links.json`) because I realized they are both JSON HTTP responses at the end of the day, so putting an extension on one implied the other was not JSON, even though it is, and blah blah blah confusing.

Reply:
I think there's a good chance we'll be glad we have this.